### PR TITLE
feat(alfred): use alfred environment variables

### PR DESCRIPTION
**Pull request by macinjoke**
ISSUES CLOSED: #118 

I updated bash script for 2 features. ( Sorry for 1 pull request )

# add `--experimental-worker` Node option automatically for Node 10. 
#118

# set node_path variables for easy setting Node path.

Alfred environment variables are useful when using user depended value like Node path.
So I set the node_path environment variable and documented how to use it.
Even if the workflow is updated, environment variables don't change. 👍

### CODE REVIEW

File: dist/workflow/info.plist [hunk 1]
```diff
@@ -162,19 +167,25 @@ node alfred-workflow-todoist.js remove "{query}"</string>
...
 ###############################################################################
 
-export PATH="$PATH:/usr/local/bin:/usr/bin"
+export PATH="$PATH:$node_path:/usr/local/bin:/usr/bin"
 
 if which node &gt; /dev/null; then
   :
 else
-  echo "You need to have node (8.5 or up) installed for this workflow to work."
+  echo "You need to have node (10.5 or up) installed for this workflow to work."
```
Comment (macinjoke) [inline dist/workflow/info.plist]: worker threads work on v10.5 or up.

**Comment by moranje** [comment]
Thanks for taking the time to do this. I'll try to walk through the changes this weekend.

Martien

**Comment by macinjoke** [comment]
@moranje ping

**Comment by moranje** [comment]
@macinjoke thanks for the ping. I'm a bit swamped at the moment, haven't had any free time to spend on OS. This is high on my priority list.

**Pull request closed without merging**